gradle/spotless.gradle

```diff
@@ -19,6 +19,12 @@ spotless {
     ktlint().userData(['indent_size': '2', 'continuation_indent_size': '2'])
     licenseHeaderFile rootProject.file('gradle/enforcement/spotless.license.java'), '(package|import|public)'
   }
+  format 'misc', {
+    target '.gitignore', '*.md', 'docs/**/*.md', '*.sh'
```

Comment:
We don't have `docs` in modules, do we? And shouldn't it be `**/*.md`?

Reply:
> We don't have `docs` in modules, do we?

not yet (and maybe never), so removed for now

> And shouldn't it be `**/*.md`?

not using `**/` patterns to keep spotless fast, added a comment here about it

Reply:
So that we will format md files only in the root of modules? Is it ok?

Reply:
interesting, we do have some md files not in the root of modules:

```
./agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/jdbc/README.md
./agent-bootstrap/src/test/resources/classloader-test-jar/testjar-readme.md
```

how about I limit it to `*.md,src/**/*.md`?

Reply:
That's good